storage/mount-filesystem.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -e  # Exit immediately if a command exits with a non-zero status
+
+# Color codes
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
```

Comment:
It is safer to use light colors on text, for example, this blue is almost invisible on my terminal

![image](https://github.com/user-attachments/assets/3b884550-efda-4719-a46f-aec3781366a8)


```suggestion
RED='\033[1;31m'
GREEN='\033[1;32m'
YELLOW='\033[1;33m'
BLUE='\033[1;34m'
NC='\033[0m' # No Color
```